jsk_fetch_robot/jsk_fetch.rosinstall

```diff
@@ -1,9 +1,9 @@
 - git:
     local-name: jsk_robot
     uri: https://github.com/jsk-ros-pkg/jsk_robot.git
-    version: 824b165
+    version: master
 - git:
     local-name: jsk_pr2eus
     uri: https://github.com/jsk-ros-pkg/jsk_pr2eus.git
-    version: 0.3.7
+    version: master
```

Comment:
https://github.com/jsk-ros-pkg/jsk_pr2eus/commits/master if there is important commit since 0.3.10, remove this line and use deb version of pr2eus